docs/quickstart.md

```diff
@@ -0,0 +1,293 @@
+
+#### Regularizer Hyperparameters
+
+Object of type regularizer have different hyper-parameters, depending on the type, but they all share the following:
+
+1. `solver_name`: The name of the [`jaxopt`](https://jaxopt.github.io/stable/) solver used for learning the model parameters (for instance, 
+    `GradientDescent`, `BFGS`, etc.)
+2. `solver_kwargs`: Additional arguments that should be passed to the solver (for instance, `tol`, `max_iter` and 
+    similar) as a dictionary.
+
+For each regularizer, we have one or more allowable solvers, stored in the `allowed_solvers` attribute. 
+This ensures that for each regularization scheme, we run an appropriate optimization algorithm. 
+
+```python
+import nemos as nmo
+
+regularizer = nmo.regularizer.Ridge()
+print(f"Allowed solver: {regularizer.allowed_solvers}")
+```
+
+!!! note
+    The `allowed_solvers` shouldn't be changed, changing the default values of this attribute will 
+    result in an exception being raised.
+
+Except for `Unregularized`, all the other `Regularizer` objects will have the `regularizer_strength` hyper-parameter.
+This is particularly helpful for controlling over-fitting. In general, the larger the `regularizer_strength` 
+the smaller the coefficients. Usually, one should tune this hyper-parameter by means of cross-validation. Look at the
+[Integration  with `scikit-learn`](#interactions-with-scikit-learn) session for a concrete example.
+
+### Interactions with `pynapple`
+
+!!! warning
+    This section assumes some familiarity with the `pynapple` package for time-series manipulation and data 
+    exploration. If you'd like to learn more about it, take a look at the [`pynapple` documentation](https://pynapple-org.github.io/pynapple/).
+
+If you represent your task variables and/or spike counts as `pynapple` time-series, don't worry, `nemos` estimators are 
+fully compatible with it. You can pass your time series direclty to the `fit` and `score` methods, as well as to 
+`predict`.
+
+In `nemos`, when a transformation preserve the time-axis, if a `pynapple` time-series is provided as input, 
+the output will be a `pynapple` time-series too!
+
+A canonical example of this behavior is the `predict` method of `GLM`. The method receives a time-series of features as 
+input  and outputs a corresponding time-series of firing rates.
+
+
+```python
+import nemos as nmo
+import numpy as np
+import pynapple as nap
+
+# predictors and observed counts as pynapple time-series with data
+time = np.arange(100)
+X = nap.TsdTensor(t=time, d=0.2 * np.random.normal(size=(100, 1, 1)))
+coef = np.random.normal(size=(1, 1))
+# compute the firing rate as a weighted sum of the feature
+firing_rate = np.exp(np.einsum("nf, tnf -> tn", coef, X))
+# generate spikes
+y = nap.TsdFrame(t=time, d=np.random.poisson(firing_rate))
+
+model = nmo.glm.GLM()
+# the following works
+model.fit(X, y)
+
+# this will return a pynapple TsdFrame
+print(type(model.predict(X)))
+```
+
+#### Why should you care?
```

Comment:
While this is a good section, I think it's too long for the quickstart. Maybe just keep the first paragraph and a half, and, once the workshop tutorials get added, link to them to show more details

Reply:
I disagree with removing the next part because I think it showcases how powerful the interaction between the packages could be. Let's have @gviejo have a saying and break even

Reply:
expanding on my point of view: one would need context to understand why this process of combining pynapple and nemos is interesting